lib/future/README.md

```diff
@@ -0,0 +1,6 @@
+# Future Library
+
+This folder is intended for future additions to the TSS and is meant as a sandbox/playground
+until the TSS is ready to take it in.
```

Comment:
I'm not sure to like this future sub-dir. Did I understand correctly that the idea of future is to have code that may eventually made to the TSS specification? Shouldn't then be added to the tpm2-tss project?

Please elaborate a little more about the goal of this and the expected process would be to push this lower in the stack or into the TCG specs.

Reply:
Right now I see a lot of code with no users in the repo, so it's kind of hard to review... I think it would be easier if these can make as normal helpers in the tpm2-tools project (living under lib like all the other helpers) and the tools be ported to it. Don't even need to be on a flag day, but can be done incrementally IMHO.

Reply:
@martinezjavier as far as in-code users, the tools where it reports the error hex code will be updated to use this. As for the goal, the goal is to get it into the spec, but since that is a ways off and the TSS project has it's own goal bigger then this at the moment, and I want it in the next release, I was proposing the future sub-dir.

With that said, I am not tightly coupled to the future dir, and we can drop that and just pull it in, adhering to our coding standard and worry about a port later.